tests/conftest.py

```diff
@@ -1,9 +1,12 @@
 import pytest
-import requests
 
 from interactive.models import User
 
 
+# include some fixtures from submodules
+pytest_plugins = ["tests.unit.services.test_opencodelists"]
```

Comment:
Does this set up the fixtures from test_opencodelists as global fixtures?

Reply:
Yes, it makes them automatically available, as if it had been defined in conftest.py